meson.build

```diff
@@ -11,6 +11,7 @@ project(
     'buildtype=debugoptimized',
     'c_std=c99',
     'cpp_std=c++14',  # TODO: use c++11 if 14 is not available
+    'cpp_eh=default',
```

Comment:
This is already the default says https://mesonbuild.com/Builtin-options.html#compiler-options, so why does it need to be added here?